Formula/ohdear-cli.rb

```diff
@@ -0,0 +1,22 @@
+class OhdearCli < Formula
+  desc "Tool to manage your Oh Dear sites"
+  homepage "https://github.com/nunomaduro/ohdear-cli"
+  url "https://github.com/nunomaduro/ohdear-cli/releases/download/v3.0.0/ohdear-cli.phar"
+  sha256 "257e792bf575277e82313350b06100f915711c939b2dc83f88099004397ff7f9"
+  license "MIT"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "php"
+
+  def install
+    bin.install "ohdear-cli.phar" => "ohdear-cli"
+  end
+
+  test do
+    system "#{bin}/ohdear-cli", "--version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
I'll add a test in for the Auth check tomorrow. 👍🏻 Cheers.

Reply:
I've updated this to `v3.1.0` (which added some better handling of unauthorised users), and updated the test to check. 👍🏻